### DISCUSSION TITLE
Save custom table views

### DISCUSSION THREAD

**Discussion by jannikmaierhoefer** [langfuse/langfuse#4657] [Ideas]
### Describe the feature or potential improvement

It would be helpful to be able to save different views that are filtered to specific tags. This would allow to quickly get visibility for traces that belong to a specific feature.

**Comment by marcklingen** [comment]
+1 on this, in the meantime I solve this by saving some views as browser bookmarks

**Comment by marliessophie** [comment]
We just launched this feature in beta and would love feedback!

**Reply by oakgary** [reply]
<img width="624" height="129" alt="Screenshot 2025-07-17 at 10 18 09" src="https://github.com/user-attachments/assets/da0fbb53-7219-4d6e-b8ab-03cae299c4ea" />

@marliessophie When I click the button to generate a permalink I get the toast that it was copied into my clipboard but my clipboard was not updated. As a user I am not sure how to share a custom view.

**Comment by ndavid-vsante** [comment]
@marliessophie Saved table view doesn't seem to take into account the selected environment. I am trying to create a view of Scores only in PROD env, but as soon as i change the Env. selection, the saved view doesn't put it back. 

I also have the same clipboard issue as oakgary.

@marcklingen The browser bookmark trick works, but it also doesn't take into account the Environment selection as there doesn't seem to be any URL param for this filter.

TBH, at first i was looking into dashboard+widgets to create custom views. I was trying to create table widgets to see a dashboard with lists of most recent traces, and list of recent user feedback, list of traces with bugs... It would be cool to create a dashboard with widgets of type Custom Table View.

Thanks your your hard work, you rock! Our team loves Langfuse!

**Comment by OumaimaSouilem18** [comment]
ça sera très utile si on peut utiliser les views crées comme source de données 'View' dans la création des widgets. Ça sera pertinent pour avoir un dashboard plus custom.
